BatteryChargingTest.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Diagnostics;
+
+namespace BatteryCharging
+{
+    public class BatteryChargingTest
+    {
+       public static void BatteryChargingTestFactors(float chargingTemperature_Celsius, float stateOfCharge_percentage, float chargeRate_CRate)
```

Comment:
Instead of Parameterizing values ...why don't you parameterize Check**** functions ?

Reply:
Could you please guide me how to proceed with this change and how changes are beneficial ?